src/main/java/chatterbox/Main.java

```diff
@@ -0,0 +1,45 @@
+package chatterbox;
+
+import java.io.IOException;
+
+import chatterbox.ui.MainWindow;
+import javafx.application.Application;
```

Comment:
I'm not so sure but should there be an empty line in between since both are from different dependencies?